pc.c

```diff
@@ -234,19 +237,20 @@ bool SSL_ItronEnhanced_CompressKey(
 {
     /* ANSI X9.62-2005 A5.7 */
 
+    mp_init_multi(&param_p, &param_a, &param_b, &three, &alpha, &beta, &u, &y, &z, &xpi, &tmp1, &ypi, NULL);
```

Comment:
this code assumes that this function will be 'always' called first. you can move this to an init function that gets called by both both compress and expand after checking an ' initialized' flag.

break this long line